docs/design/cincinnati.md

```diff
@@ -39,6 +39,7 @@ An example of this metadata document can be seen here:
 
 ```json
 {
+	"kind": "cincinnati-metadata-v0",
```

Comment:
@crawford does this belong here? The text above says it's required

Reply:
I think the `schema` integer became the type-tag `kind` on some later iteration of this design.

Reply:
It's required as it won't deserialize without it: 

https://github.com/openshift/cincinnati/blob/95db15a823b231636c7a3aaeef25b6c22e6c9a8b/graph-builder/src/release.rs#L22

https://github.com/openshift/cincinnati/blob/95db15a823b231636c7a3aaeef25b6c22e6c9a8b/graph-builder/src/release.rs#L47-L50